README.md

```diff
@@ -12,3 +12,53 @@ definitions.
  - The spec generator that is responsible for dynamically generate 
 the final OpenAPI spec using web service routes or combining other 
 OpenAPI/Json specs.
+
+## Development
+
+After cloning this repo, within the top-level directory do the following to
+resolve and download dependencies:
+
+```
+$ go get -t -v ./...
+```
+
+### Unit testing
+
+From top-level directory:
+
+```
+$ go test ./pkg/...
+```
+
+### Testing with Kubernetes
+
+Assuming you already have a `k8s.io/kubernetes` repo, and `k8s.io/kube-openapi` repo:
+
+* Remove vendored `kube-openapi` within `k8s.io/kubernetes`. From top-level directory:
```

Comment:
I would prefer to make this not necessary than document it :)

Reply:
Agreed. lets remove it from documentation and add an issue in the repo to fix it.